anvil/config.py

```diff
@@ -27,34 +27,37 @@ class ConfigParser(Config):
     objects
     """
 
-    def parse_lattice_length(self, length: int):
```

Comment:
Can you please also fix the imports at top of this file?

standards, third party, reportengine then anvil each with their own block.